autoscale-api/src/test/java/com/sequenceiq/periscope/api/endpoint/validator/DistroXAutoscaleRequestValidatorTest.java

```diff
@@ -38,8 +43,12 @@
 
     private ConstraintValidatorContext validatorContext;
 
+    @Mock
+    private CloudbreakMessagesService messagesService;
+
     @Before
     public void setupMocks() {
+        MockitoAnnotations.initMocks(this);
```

Comment:
this should work without this because the class is annotated with `@RunWith(MockitoJUnitRunner.class)` or am I missing something?

Reply:
yes this is not required, fixed it.